src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -266,10 +302,14 @@ public IEnumerable<FrameworkSpecificMapping> EquivalentProfiles
                     _subSetFrameworks = new KeyValuePair<string, string>[]
                         {
                             // .NET is a subset of DNX
-                            new KeyValuePair<string, string>(FrameworkConstants.FrameworkIdentifiers.Net, FrameworkConstants.FrameworkIdentifiers.Dnx),
-
-                            // DotNet is a subset of DNXCore
-                            new KeyValuePair<string, string>(FrameworkConstants.FrameworkIdentifiers.NetPlatform, FrameworkConstants.FrameworkIdentifiers.DnxCore),
+                            new KeyValuePair<string, string>(
+                                FrameworkConstants.FrameworkIdentifiers.Net,
+                                FrameworkConstants.FrameworkIdentifiers.Dnx),
+
+                            // NetStandard is a subset of DNXCore
+                            new KeyValuePair<string, string>(
```

Comment:
I believe this is fixed on [my latest iteration](https://github.com/NuGet/NuGet.Client/blob/jver-1691/src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs#L272) -- correct me if I am missing something.